### ISSUE TITLE
other server.properties

### ISSUE THREAD

**Issue by hypergig** [itzg/docker-minecraft-bedrock-server#263]
Is there a way to adjust other `server.properties` such as pvp? Is there some prior at for overriding the whole file with a config map for example?

**Comment by mattholy** [comment]
It’s easy to modify the file directly if you mounted it
outside the container

在 2022年7月10日，22:14，Jordan Cohen ***@***.***> 写道：

﻿

Is there a way to adjust other server.properties such as pvp? Is there some prior at for overriding the whole file with a config map for example?

**Comment by hypergig** [comment]
> It’s easy to modify the file directly if you mounted it outside the container 在 2022年7月10日，22:14，Jordan Cohen ***@***.***> 写道： ﻿ Is there a way to adjust other server.properties such as pvp? Is there some prior at for overriding the whole file with a config map for example? — Reply to this email directly, view it on GitHub<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fitzg%2Fdocker-minecraft-bedrock-server%2Fissues%2F263&data=05%7C01%7C%7C4ec4de1b4c814bfe8c1608da627e8835%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637930592858541547%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=c2jixKekEZcR1hM0ifG9sQ37fFR6Z6tqCJjU0pwCW3I%3D&reserved=0>, or unsubscribe<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFJGQ2PMNISMKFUMHETCMUTVTLLFBANCNFSM53FCVFNA&data=05%7C01%7C%7C4ec4de1b4c814bfe8c1608da627e8835%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637930592858541547%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Av8tTQFzhC2HZkOk4Mdg3aeIPK%2B8RIAGmWTUO5nF2z8%3D&reserved=0>. You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Yeah this is what I have been doing but i was interested in doing it via automation. I am interested in spinning up multiple arbitrary servers, and was thinking of writing a kubernetes minecraft operator.

**Comment by mattholy** [comment]
Set environment variables then, you can view the init script for your needs

在 2022年7月11日，00:39，Jordan Cohen ***@***.***> 写道：

﻿

It’s easy to modify the file directly if you mounted it outside the container 在 2022年7月10日，22:14，Jordan Cohen @.> 写道： ﻿ Is there a way to adjust other server.properties such as pvp? Is there some prior at for overriding the whole file with a config map for example? — Reply to this email directly, view it on GitHubhttps://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fitzg%2Fdocker-minecraft-bedrock-server%2Fissues%2F263&data=05%7C01%7C%7C4ec4de1b4c814bfe8c1608da627e8835%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637930592858541547%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=c2jixKekEZcR1hM0ifG9sQ37fFR6Z6tqCJjU0pwCW3I%3D&reserved=0<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fitzg%2Fdocker-minecraft-bedrock-server%2Fissues%2F263&data=05%7C01%7C%7C03c3cc436ffb483666f008da6292c0a4%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637930679695251862%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=8E46Y26X7SKwiQSXqMaw3uOGmlukOdEH19O4pFUnHLI%3D&reserved=0>, or unsubscribehttps://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFJGQ2PMNISMKFUMHETCMUTVTLLFBANCNFSM53FCVFNA&data=05%7C01%7C%7C4ec4de1b4c814bfe8c1608da627e8835%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637930592858541547%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Av8tTQFzhC2HZkOk4Mdg3aeIPK%2B8RIAGmWTUO5nF2z8%3D&reserved=0<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAFJGQ2PMNISMKFUMHETCMUTVTLLFBANCNFSM53FCVFNA&data=05%7C01%7C%7C03c3cc436ffb483666f008da6292c0a4%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637930679695251862%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=BAU2t6GWmOc9I8XpjdBXc1oYRzRosVSzkHN3OopLj78%3D&reserved=0>. You are receiving this because you are subscribed to this thread.Message ID: @.>

Yeah this is what I have been doing but i was interested in doing it via automation. I am interested in spinning up multiple arbitrary servers, and was thinking of writing a kubernetes minecraft operator.

**Comment by BaksiLi** [comment]
Run the docker first, then in CLI (or in GUI -> Containers, you can find a little terminal icon), enter:
```
docker exec <CONTAINER_NAME_OR_ID> /bin/sh
```

And you should be able to access `server.properties`.

If you do this frequently and do not want to mount the volume, you can install vim inside the container `apt install vim`.
